cli/src/cli_util.rs

```diff
@@ -3298,3 +3298,48 @@ fn format_template_aliases_hint(template_aliases: &TemplateAliasesMap) -> String
     );
     hint
 }
+
+pub fn parse_author_arg(author: &str) -> Result<(String, String), clap::Error> {
+    let re = regex::Regex::new(r"(?<name>.*?)\s*<(?<email>.+)>").unwrap();
```

Comment:
needs `$` anchor to not discard trailing text?

I think using `.rsplit_once()`, `.strip_suffix()`, `.trim_ascii()`, etc. is also good.

Reply:
`$` makes sense, if there is something after `>` then it's improper syntax.
Added.